app/controllers/exports_controller.rb

```diff
@@ -93,6 +93,6 @@ def locate_ancestor(plate_id)
   end
 
   def include_parameters
-    csv_details.plate_includes || 'wells'
+    csv_details.labware_includes || 'wells'
```

Comment:
These should probably still be plate includes, as they get used as part of: `Sequencescape::Api::V2.plate_with_custom_includes` so don't currently support tubes. (Although in turn this is wrapped in locate_labware so the code isn't exactly honest about its limitations as is)